examples/experimental/dagster-airlift/dagster_airlift/core/utils.py

```diff
@@ -64,3 +68,10 @@ def task_handles_for_spec(spec: AssetSpec) -> Set["TaskHandle"]:
             TaskHandle(dag_id=task_handle_dict["dag_id"], task_id=task_handle_dict["task_id"])
         )
     return set(task_handles)
+
+
+def maps_to_dag(asset_spec: AssetSpec, dag_id: str) -> bool:
+    return (
+        STANDALONE_DAG_ID_METADATA_KEY in asset_spec.metadata
+        and asset_spec.metadata[STANDALONE_DAG_ID_METADATA_KEY] == dag_id
+    )
```

Comment:
I think this name is too general as people might think that it means that it maps to any _task_ in that dag.

We need consistent terminology for the "dag assets" and to distinguish them from the mapped external assets.